test/test.js

```diff
@@ -554,4 +562,440 @@ describe('parser (exif data)', () => {
+			assert.equal(exif.flir.AtmosphericTransBeta1, -0.00227600010111928)
+			assert.equal(exif.flir.AtmosphericTransBeta2, -0.006670000031590462)
+			assert.equal(exif.flir.AtmosphericTransX, 1.899999976158142)
+			assert.equal(exif.flir.CameraTemperatureRangeMax, 408.1499938964844)
+			assert.equal(exif.flir.CameraTemperatureRangeMin, 248.14999389648438)
+			assert.equal(exif.flir.UnknownTemperature1, 408.1499938964844)
+			assert.equal(exif.flir.UnknownTemperature2, 228.14999389648438)
+			assert.equal(exif.flir.UnknownTemperature3, 408.1499938964844)
+			assert.equal(exif.flir.UnknownTemperature4, 248.14999389648438)
+			assert.equal(exif.flir.UnknownTemperature5, 423.1499938964844)
+			assert.equal(exif.flir.UnknownTemperature6, 223.14999389648438)
+			assert.equal(exif.flir.CameraModel, 'FLIR Tau 2 640')
+			assert.equal(exif.flir.CameraPartNumber, '46640019H-SRNLX')
+			assert.equal(exif.flir.CameraSerialNumber, '148722')
+			assert.equal(exif.flir.CameraSoftware, '')
+			assert.equal(exif.flir.LensModel, 'FOL19')
+			assert.equal(exif.flir.LensPartNumber, '*')
+			assert.equal(exif.flir.LensSerialNumber, '*')
+			assert.equal(exif.flir.FieldOfView, 0)
+			assert.equal(exif.flir.FilterModel, '')
+			assert.equal(exif.flir.FilterPartNumber, '')
+			assert.equal(exif.flir.FilterSerialNumber, '')
+			assert.equal(exif.flir.PlanckO, -56)
+			assert.equal(exif.flir.PlanckR2, 1)
+			assert.equal(exif.flir.RawValueMedian, 4458)
+			assert.equal(exif.flir.RawValueRange, 1805)
+			assert.equal(exif.flir.FocusStepCount, 0)
+			assert.equal(exif.flir.FocusDistance, 0)
+			assert.equal(exif.flir.PaletteName, 'Rainbow')
+			assert.equal(exif.flir.GPSMapDatum, 'WGS-84')
+		})
+
+		it('Sensefly Thermomap 9mm', async () => {
+			const exif = await parse(buffers['Sensefly_Thermomap_9mm.tif'], options)
+			assert.exists(exif, `exif doesn't exist`)
+			assert.equal(exif.ProcessingSoftware, 'eBee')
+			assert.equal(exif.ImageWidth, 640)
+			assert.equal(exif.ImageHeight, 512)
+			assert.equal(exif.BitsPerSample, 16)
+			assert.equal(exif.Compression, 5)
+			assert.equal(exif.Make, 'senseFly')
+			assert.equal(exif.Model, 'thermoMAP')
+			assert.equal(exif.StripOffsets, 3012)
+			assert.equal(exif.SamplesPerPixel, 1)
+			assert.equal(exif.StripByteCounts, 518880)
+			assert.deepEqual(exif.DateTimeOriginal, new Date('2015-11-19T10:35:28.000Z'))
+			assert.equal(exif.FocalLength, 9)
+			assert.equal(exif.GPSLatitudeRef, 'N')
+			assert.deepEqual(exif.GPSLatitude, [ 40.0594223, 0, 0 ])
+			assert.equal(exif.GPSLongitudeRef, 'E')
+			assert.deepEqual(exif.GPSLongitude, [ 23.402305, 0, 0 ])
+			assert.equal(exif.GPSAltitude, 142.365088)
+			assert.equal(exif.latitude, 40.0594223)
+			assert.equal(exif.longitude, 23.402305)
+
+			// XMP
+			assert.exists(exif.xmp)
+		})
+
+		it('Wiris 2nd Gen 640 19mm', async () => {
+			const exif = await parse(buffers['Wiris_2nd_Gen_640_19mm.tiff'], options)
+			assert.exists(exif, `exif doesn't exist`)
+			// TODO: Unknown tags
```

Comment:
The Wiris has some unknown tags that even ExifTool doesn't parse. They contain the letters `N` and `E`, so they seem to indicate some kind of wind direction, but I couldn't find anything about it online.

Reply:
N and E are GPS values (northig easting)

Reply:
Do you know how the tags should be called then?

Reply:
Something as GPSLatitude/Longitude Ref https://sno.phy.queensu.ca/~phil/exiftool/TagNames/GPS.html

Reply:
Something as GPSLatitude/Longitude Ref https://sno.phy.queensu.ca/~phil/exiftool/TagNames/GPS.html

Reply:
but they should already exist

Reply:
It's weird that those are the only GPS tags it can't parse. Is it ok if I add them to the list of tags?

Reply:
we can add them. dont find antything about it in exiftool:?

Reply:
I didn't find anything about the specific hex tag...

Reply:
I added the tags with a comment